### ISSUE TITLE
Morning session code review

### ISSUE THREAD

**Issue by erickma1** [AlselwiAisha/Aisha_and_Qudri#3]
Very good work with your html structure. I may suggest the script tags in the index.html file could have been placed before the body tag

**Comment by abujafar93** [comment]
Yes. That's the best practice. It should be placed in the head tag with
defer or async attribute when src is used. Thanks for that.
